src/lib/omerlo/reader/index.ts

```diff
@@ -4,15 +4,21 @@ import { parseContentBlockTemplate, parseContentTemplate } from './endpoints/con
 import { parseProfileTypeSummary } from './endpoints/profileType';
 import { registerAssocParser } from './utils/assocs';
 import { parseContentSummary } from './endpoints/contents';
+import { parseIssueBlockConfiguration, parseIssueType } from './endpoints/magazines';
 
 export * from './stores/user_session';
 export type * from './endpoints/oauth';
 export type * from './endpoints/accounts';
 
 registerAssocParser('categories', parseCategory);
 registerAssocParser('profiles', parseProfileSummary);
-registerAssocParser('templates', parseContentTemplate);
+registerAssocParser('content_templates', parseContentTemplate);
 registerAssocParser('profile_types', parseProfileTypeSummary);
 registerAssocParser('profile_block_types', parseProfileBlock);
 registerAssocParser('block_templates', parseContentBlockTemplate);
```

Comment:
```suggestion
registerAssocParser('content_block_templates', parseContentBlockTemplate);
```

I changed that last week ;)